src/pages/python/powxy/index.md

```diff
@@ -21,4 +21,4 @@ If `z` is present it returns `x` to the power `y` modulo `z`.If only `x` and `y`
 
 ![:rocket:](//forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=2 ":rocket:") <a href='https://repl.it/CTGi' target='_blank' rel='nofollow'>Run Code</a>
```

Comment:
This image doesn't display when you preview the article. This needs to be fixed too.